src/ucp/proto/proto_select.c

```diff
@@ -268,6 +267,15 @@ ucp_proto_select_init_protocols(ucp_worker_h worker,
             continue;
         }
 
+        if (init_params.ep_config_key->err_mode != UCP_ERR_HANDLING_MODE_NONE) {
+            ucs_assertv(ucp_protocols[proto_id]->abort !=
+                        ucp_proto_abort_fatal_not_implemented,
+                        "proto %s was selected for err handling mode %d with "
+                        "missed abort implementation",
```

Comment:
what if abort() is not really needed? maybe the protocol may never adds anything to pending queue

Reply:
I think it should be initialized with empty function stub

Reply:
ok
so pls change it to
"selected protocol %s doesn't implement abort() function, but err handling mode is %d"